src/components/accordion/Accordion.tsx

```diff
@@ -66,17 +69,17 @@ function Accordion({
         </>
       ))}
       <ShowMoreLabel
-        show={!showFullList && items.length > itemsCutoff}
+        show={collapsed && items.length > itemsCutoff}
         items={items.slice(itemsCutoff)}
-        toggle={() => setShowFullList(!showFullList)}
+        toggle={() => setCollapsed(!collapsed)}
       />
     </ul>
   )
 }
 
 export function ShowMoreLabel(props: ShowMoreLabelProps) {
   if (!props.show) return null
-  return <span className="whitespace-pre text-xs font-medium opacity-70 hover:cursor-pointer" onClick={props.toggle}>+{props.items.length} day(s) more</span>
+  return <span className="whitespace-pre text-xs font-medium opacity-70 hover:cursor-pointer" onClick={props.toggle}>+{props.items.length} day{props.items.length > 1 ? "s" : ""} more</span>
```

Comment:
You need to switch it to "x more days" instead of "x days more" i think.

Reply:
Ahhh, that was what you meant :D